Makefile

```diff
@@ -222,7 +222,7 @@ github-token.txt:
 .PHONY: buildx
 buildx:
 buildx:
-	 docker buildx create --driver=docker-container --use --name=konvoy-image-builder || true
+	 docker buildx create --use --name=konvoy-image-builder || true
```

Comment:
Why was this change needed?

Reply:
It's not needed, but it makes things a lot faster because the cache can be re-used

Reply:
I can pull it out to separate PR if you'd like

Reply:
Yes, that would be good.

Reply:
This was added when we were trying arm64 support. We later find out that we dont need it. This argument `--driver=docker-container` unnecessary slows builds down and does not cache docker file system layers between builds.

Reply:
removed and will file another PR